samza-yarn/src/main/java/org/apache/samza/job/yarn/YarnFaultDomainManager.java

```diff
@@ -117,4 +120,8 @@ public boolean hasSameFaultDomains(String host1, String host2) {
     }
     return hostToRackMap;
   }
+
+  private void initMetrics() {
+    hostToFaultDomainCacheUpdates = metricsRegistry.newCounter(groupName, "host-to-fault-domain-cache-updates");
+  }
```

Comment:
I extracted it just in case we add other metrics in the future. Having said that, I don't see any other metrics being needed as of now, so have inlined.